feat(changelog): billing-added-billing-api- 2023-01-19 (#1217)
scaleway/docs-content: changelog/january2023/2023-01-19-billing-added-billing-api-.mdx

---

> title: Billing API 

Reviewer: ```suggestion
title: The billing API is now available!
```

status: added
author:
    fullname: 'Join the #community channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-01-19
category: billing
product: billing
---

> Billing API is now available, you can now retrieve your current consumption and invoices via API. Follow our   <Link to=https://developers.scaleway.com/en/products/billing/api/>developer documentation here</Link>

Reviewer 2: ```suggestion
Billing API is now available, you can now retrieve your current consumption and invoices via API. Follow our [developer documentation here](https://developers.scaleway.com/en/producs/billing/api/).
```

> Billing API is now available, you can now retrieve your current consumption and invoices via API. Follow our   <Link to=https://developers.scaleway.com/en/products/billing/api/>developer documentation here</Link>

Reviewer: ```suggestion
With the billing API, you can now retrieve your current consumption and invoices via API. Follow our   <Link to=https://developers.scaleway.com/en/products/billing/api/>developer documentation here</Link>
```
